4/Step 1/Shipment.ts

```diff
@@ -0,0 +1,53 @@
+import {Shipper} from "../Step 2/Shipper";
+
+export interface ShipmentInfo {
+    ShipmentID: number;
+    Weight: number;
+    FromAddress: string;
+    FromZipCode: string;
+    ToAddress: string;
+    ToZipCode: string;
+}
+
+const PRICE_PER_OUNCE = 0.39;
+
+export class Shipment {
+    public static id = 0;
```

Comment:
This should not be `public`. And why is it `static`? Is it used anywhere else? If yes, you should encapsulate it and allow access only by methods.